The Debian alpha buildd runs out of fds with this ulimit in place.  It's
undiagnosed why currently, but the GCC compilefarm alpha box is down and
and I got no response on #debian-alpha or #debian-devel.  The ulimit is
really to make sure we catch a new fd leak on our dev boxes rather than
when a user on OpenBSD or similar hits it, so just disable it on alpha
for now.
Index: xapian-core/tests/runtest.in
===================================================================
--- xapian-core.orig/tests/runtest.in	2009-08-05 02:58:13.000000000 +1200
+++ xapian-core/tests/runtest.in	2009-08-05 03:04:34.000000000 +1200
@@ -35,7 +35,18 @@
 # If ulimit supports it, limit the number of open file descriptors so we catch
 # file descriptor leaks sooner.  Not all platforms support changing this limit
 # but Linux does at least.
-ulimit -n 64 2>/dev/null || true
+case `uname -s -m` in
+"Linux alpha")
+  # The Debian alpha buildd runs out of fds with this ulimit in place.  It's
+  # undiagnosed why currently, but the GCC compilefarm alpha box is down and
+  # and I got no response on #debian-alpha or #debian-devel.  The ulimit is
+  # really to make sure we catch a new fd leak on our dev boxes rather than
+  # when a user on OpenBSD or similar hits it, so just disable it on alpha
+  # for now.
+  ;;
+*)
+  ulimit -n 64 2>/dev/null || true ;;
+esac
 
 case $1 in
   *test|*test@EXEEXT@) ;;
